src/simulation.hpp

```diff
@@ -942,6 +969,9 @@ void AMRSimulation<problem_t>::MakeNewLevelFromScratch(
   // set state_new_[lev] to desired initial condition
   setInitialConditionsAtLevel(level);
 
+  amrex::IntVect iv(AMREX_D_DECL(19, 0, 5));
+  print_state(state_new_[level], iv);
+
```

Comment:
Delete this debugging code.

Reply:
Ugh, I thought I had already deleted this. Done now (again).